documentation/features/set-volume.mdx

```diff
@@ -0,0 +1,13 @@
+# Set Volume
+
+This is a android only API. It allows you to vary the volume level of the remote peer individually for yourself. 
```

Comment:
pls remove android only

Reply:
It allows you to locally vary the volume level  of the remote peer's audio track.